app/components/Configuration.tsx

```diff
@@ -1,49 +1,43 @@
-// @flow
-import type { Node } from 'react';
-import React, { Component } from 'react';
-import styles from './Configuration.scss';
-import type {
-  configurationBoolean,
-  configurationNumbers
-} from '../reducers/configuration';
-import { type Configuration as ConfigurationType } from '../reducers/configuration';
-import stages from './configurationStages';
-import type {
-  FlexibleNode,
-  StageDescription
-} from './configurationStages/types';
-
+import type { Node } from "react";
+import React, { Component } from "react";
+import styles from "./Configuration.scss";
+import type { configurationBoolean, configurationNumbers } from "../reducers/configuration";
+import type { Configuration as ConfigurationType } from "../reducers/configuration";
+import "../reducers/configuration";
+import stages from "./configurationStages";
+import type { FlexibleNode, StageDescription } from "./configurationStages/types";
 type Props = {
-  nextStage: () => void,
-  previousStage: () => void,
-  hideConfiguration: () => void,
-  togglePostcode: (postcode: string) => void,
-  resetPostcodes: () => void,
-  toggleFloor: (floor: number) => void,
-  toggleBoolean: (name: configurationBoolean) => void,
-  setNumber: (name: configurationNumbers, value: ?number) => void,
-  setString: (name: string, value: ?string) => void,
-  resetConfiguration: () => void,
-  configuration: ConfigurationType
+  nextStage: () => void;
+  previousStage: () => void;
+  hideConfiguration: () => void;
+  togglePostcode: (postcode: string) => void;
+  resetPostcodes: () => void;
+  toggleFloor: (floor: number) => void;
+  toggleBoolean: (name: configurationBoolean) => void;
+  setNumber: (name: configurationNumbers, value: number | null | undefined) => void;
```

Comment:
i'm confused that both `null` and `undefined` are allowed here, have you checked if this makes sense where this call ends up?

Reply:
yes, for the setNumber there it should be only null as I see it from https://github.com/neopostmodern/wohnungsbot/blob/dev/app/components/inputs/NumberField.js#L3

for the setString (one line below) I'm confused and cannot judge if we can narrow it down